package.json

```diff
@@ -24,15 +24,15 @@
   "license": "ISC",
   "dependencies": {
     "lodash": "4.8.1",
-    "react": "0.14.8"
+    "react": "0.14.8",
```

Comment:
In #17 we reduce to `14.7`, want to rebase this PR then to get it up to date so we can merge 17 first?